packages/frontend/src/ui/deck/tl-column.vue

```diff
@@ -8,12 +8,12 @@
 		<span style="margin-left: 8px;">{{ column.name }}</span>
 	</template>
 
-	<div v-if="disabled" :class="$style.disabled">
+	<div v-if="(((column.tl === 'local' || column.tl === 'social') && !isLocalTimelineAvailable) || (column.tl === 'global' && !isGlobalTimelineAvailable))" :class="$style.disabled">
 		<p :class="$style.disabledTitle">
-			<i class="ti ti-minus-circle"></i>
-			{{ $t('disabled-timeline.title') }}
+			<i class="ti ti-minus"></i>
```

Comment:
ここの変更意図を教えてもらえますか？

Reply:
もともとFont Awesomeのfa-minus-circleだったのですが、v13更新時にThemify Iconsに置き換えられています。しかしながらThemify Iconsには`ti-minus-circle`なるものはなく、表示できないので`ti-minus`に置き換えました。

Reply:
ありがとうございます。しかしながら v13 で使っているアイコンセットは Themify Icons ではなく Tabler Icons なので https://tabler-icons.io/i/circle-minus が　replacement としてより適切だと思います。

```suggestion
			<i class="ti ti-circle-minus"></i>
```

Reply:
勘違いしてました、ありがとうございます。